magento/pwa-studio
# [PWA-1514] Cannot place order again after initial order was declined

## Description

When placing an order, if the initial order is declined and cause a Checkout Error, we won't be able to try again without reloading the page.

### Steps to reproduce the behavior:

1. Create an order that will be de declined.
In our case, we are using Braintree in sandbox mode and had an order between 2000.00 - 2999.99.
https://developers.braintreepayments.com/reference/general/testing/php#transaction-amounts
2. Place order and get the Checkout Error stating your order was declined.
3. Update your payment information.
4. Place order and see that nothing happens.

### Expected behavior

We should be able to place a new order when our first try was declined.

## Related Issue





Closes [#3046](https://github.com/magento/pwa-studio/issues/3046)
[[PWA-1514](https://jira.corp.magento.com/browse/PWA-1514)] Cannot place order again after initial order was declined #3091

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Create an order that will be de declined.
In our case, we are using Braintree in sandbox mode and had an order between 2000.00 - 2999.99.
https://developers.braintreepayments.com/reference/general/testing/php#transaction-amounts
2. Place order and get the Checkout Error stating your order was declined.
4. Update your payment information.
5. Place order and see that it works.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
